Export rand

A crate that uses a different version of `rand` than this crate, say v0.5, can't use the "rand" features.
The error, `error[E0277]: the trait bound `rand::ThreadRng: rand::Rng` is not satisfied` seems to be caused by using different versions. 
One way around this is to export `rand` from `secp256k1` so that users can do e.g. `secp.generate_keypair(&mut secp256k1::rand::thread_rng())`

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -1,7 +1,7 @@
 [package]
 
 name = "secp256k1"
-version = "0.11.4"
+version = "0.11.5"
 authors = [ "Dawid Ciężarkiewicz <dpc@ucore.info>",
             "Andrew Poelstra <apoelstra@wpsoftware.net>" ]
 license = "CC0-1.0"

--- a/src/lib.rs
+++ b/src/lib.rs
@@ -135,11 +135,11 @@
 
 #![cfg_attr(all(test, feature = "unstable"), feature(test))]
 #[cfg(all(test, feature = "unstable"))] extern crate test;
-#[cfg(any(test, feature = "rand"))] extern crate rand;
-#[cfg(feature = "serde")] extern crate serde;
+#[cfg(any(test, feature = "rand"))] pub extern crate rand;
+#[cfg(feature = "serde")] pub extern crate serde;
 #[cfg(all(test, feature = "serde"))] extern crate serde_test;
 
-extern crate libc;
+pub extern crate libc;
 
 use libc::size_t;
 use std::{error, fmt, ops, ptr, str};

